web/src/app/util/SetFavoriteButton.tsx

```diff
@@ -13,14 +14,17 @@ interface SetFavoriteButtonProps {
   loading?: boolean
 }
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme: Theme) => ({
   favorited: {
-    color: 'rgb(205, 24, 49)',
+    color:
+      theme.palette.mode === 'dark'
+        ? theme.palette.primary.main
```

Comment:
I think we should maintain keeping the heart red (just what makes sense to me). Could we use `theme.palette.error` for an accessible red? Or maybe we could use a star icon instead and use `theme.palette.warn` for an accessible yellow on both light and dark modes.

Reply:
I'm open to that if it works -- firefox has a way to simulate under the Accessibility tab in the dev tools -- if you want to give it a try and post the results

Otherwise, bear in mind this is only interim and not meant to be a decision on permanent colors, just something that resolves current state while we discuss and decide next steps